### DISCUSSION TITLE
Session recording retention period

### DISCUSSION THREAD

**Discussion by alwaysastudent** [gravitational/teleport#36023] [Q&A] [answered]
Hi there, how can we control the retention period for the session recordings? Does the `retention_period` on the storage configuration apply to both the audit events as well as the session recordings? Also how can we do storage capacity planning for session recordings and audit events?

**Comment by stevenGravy** [comment, marked as answer]
`retention_period` refers to audit log entries related to DynamoDB backend usage. Session recordings are not removed.  I would expect you'd want to review the list of s3 session files and purge/move as you see fit. You could also mark them deep glacier.  

Per sizing I'd do some review of the avg size for audit records/sessions. Then multiple that for your use.
